feg/gateway/services/testcore/pcrf/mock_pcrf/mock_driven_pcrf_test.go

```diff
@@ -297,3 +300,81 @@ func getMockReAuthHandler() gx.PolicyReAuthHandler {
 		}
 	}
 }
+
+// equalRuleDefinitionSlices compares two slices with *fegprotos.RuleDefinition message pointers. Slices can be unsorted.
+// If duplicates rules exist, each has to appear the same number of times in both lists and the rule names must differ.
```

Comment:
Duplicate tolerance isn't needed here, we learned in https://github.com/magma/magma/pull/12151#discussion_r846355418. Not sure about unsorted though. May be @uri200 can help?

Also deleted the line "If duplicates exist... " since it is a bit confusing.